### ISSUE TITLE
Case History Landing Page

### ISSUE THREAD

**Issue by lheagy** [ubcgif/em#300]
The index page for the case histories would benefit from being more visual and having more info on each case history present. 
Perhaps this is accomplished in an manner similar to the contributors page (using the contributors.json)

![image](https://cloud.githubusercontent.com/assets/6361812/18359427/bee38724-75ae-11e6-857a-9964f9cf7381.png)

The casehistories.json (from which the page could be auto-generated) could be:
```json
{
    "mtisa":{
        "title":"Mt. Isa",
        "application":"Mining",
        "thumbnail":"mtisa.png",
        "citation":"rutley2001",
        "categories":["DC", "IP"],
        "contributors":{
            "authors": ["fourndo", "krisdavis"],
            "reviewers": "dwo"
    }
}
```

What is nice about having data structured like this is that eventually, we can also query by any of the fields (eg. categories, applications, etc). In addition, the html index page would be auto generated, so it could be improved independent of adding case histories.

**Comment by lheagy** [comment]
After chatting with @rowanc1, a better format might be 

```json
{
    "mtisa":{
        "title":"Mt. Isa",
        "thumbnail":"mtisa.png",
        "citations":["rutley2001"],
        "contributors":[
            {"uid":"fourndo", "role":"author"}, 
            {"uid":"kd", "role":"author"},  
            {"uid":"dwo", "role":"reviewer"}
        ],
        "tags":[
            {"uid":"DC", "as":"category"},
            {"uid":"IP", "as":"category"}, 
            {"uid":"Mining", "as":"application"}
        ]
    }
}
```

This is essentially building a graph database (we will be able to sort and visualize case histories that are tagged as being mining applications or in the category of DC)

**Comment by lheagy** [comment]
two additions: keywords should be included, as should the directory containing the case history
{
    "mtisa":{
        "title":"Mt. Isa",
        "directory":"./mtisa/",
        "thumbnail":"mtisa.png",
        "citations":["rutley2001"],
        "contributors":[
            {"uid":"fourndo", "role":"author"}, 
            {"uid":"krisdavis", "role":"author"},  
            {"uid":"dwo", "role":"reviewer"}
        ],
        "tags":[
            {"uid":"DC", "as":"category"},
            {"uid":"IP", "as":"category"}, 
            {"uid":"Mining", "as":"application", "keywords":["black shale"]}
        ]
    }
}

thoughts @rowanc1 ?

**Comment by lheagy** [comment]
closed in #332
